python-squid/src/test/java/org/sonar/python/metrics/FileMetricsTest.java

```diff
@@ -61,7 +65,14 @@ public void function_complexities() {
   private static FileMetrics metrics(String fileName) {
     File baseDir = new File("src/test/resources/metrics/");
     File file = new File(baseDir, fileName);
-    return new FileMetrics(TestPythonVisitorRunner.createContext(file), true);
+    String fileContent;
+    try {
+      fileContent = Files.toString(file, StandardCharsets.UTF_8);
+    } catch (IOException e) {
+      throw new IllegalStateException(e);
+    }
+    PyFile pyFile = new org.sonar.python.frontend.PythonParser().parse(fileContent);
```

Comment:
Now that the 7 lines above are duplicated in 3 tests in python-squid, maybe we can factorize them.

Reply:
@pynicolas https://github.com/SonarSource/sonar-python/pull/254/commits/94159450a1bde59907fd4faa7d46f3f8c443167f.